tensorflow/python/ops/special_math_ops.py

```diff
@@ -320,15 +320,15 @@ def sort_key(input_index, a):
 
     t0_shape = _get_shape(t0)
     num_broadcast_elements_t0 = _total_size(
-      t0_shape[len(preserved_axes):-len(axes_to_sum)] + [1])
-    num_summed_elements = _total_size( t0_shape[-len(axes_to_sum):])
+      t0_shape[len(preserved_axes):-len(axes_to_sum)])
+    num_summed_elements = _total_size(t0_shape[-len(axes_to_sum):])
     new_shape = (t0_shape[:len(preserved_axes)]
                  + [num_broadcast_elements_t0, num_summed_elements])
     t0 = _reshape_if_necessary(t0, new_shape)
 
     t1_shape = _get_shape(t1)
     num_broadcast_elements_t1 = _total_size(
-        t1_shape[len(preserved_axes)+len(axes_to_sum):] + [1])
+      t1_shape[len(preserved_axes)+len(axes_to_sum):])
```

Comment:
not that I know of @yifeif might know more about possible pylint options.

We use two-space indent but 4-space continuations, that's about all I know :)

Thanks for doing this